src/lib/components/Treebank/Sentence/Sentence.js

```diff
@@ -28,6 +38,8 @@ const WrappedSentence = ({
     [activeId, setActiveId] = useState(null);
   }
 
+  const highlighted = new Set(highlightedIterable);
```

Comment:
The `highlighted` being passed in can be either an array or a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set). But we convert it to a set to make lookup more efficient (see comment on `Text.js`).